packages/react-server-cli/src/coreCssMiddleware.js

```diff
@@ -0,0 +1,13 @@
+
+module.exports = function(pathToStatic) {
```

Comment:
It bothers me a little that these modules export functions that _produce_ middleware instead of just the middleware classes themselves.  I wonder if there's another way we could get `pathToStatic` in here.

Would it be terrible to just make it a global server-side also?  Already doing that browser-side...